From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andras Becsi <andras.becsi@digia.com>
Date: Mon, 24 Mar 2014 18:36:43 +0100
Subject: Hide the definition of MessagePumpX11::GetDefaultXDisplay

This should replace patch 0011 since we do not build GTK code any more.

Change-Id: I156f164128b04eee6d5956a576adef3b9d2d18ea
Reviewed-by: Jocelyn Turcotte <jocelyn.turcotte@digia.com>
---
 base/message_loop/message_pump_x11.cc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/base/message_loop/message_pump_x11.cc b/base/message_loop/message_pump_x11.cc
index 35dcc04..001775c 100644
--- a/base/message_loop/message_pump_x11.cc
+++ b/base/message_loop/message_pump_x11.cc
@@ -141,12 +141,15 @@ MessagePumpX11::~MessagePumpX11() {
   g_xdisplay = NULL;
 }
 
+#if !defined(TOOLKIT_QT)
+// Qt uses it's own version.
 // static
 Display* MessagePumpX11::GetDefaultXDisplay() {
   if (!g_xdisplay)
     g_xdisplay = XOpenDisplay(NULL);
   return g_xdisplay;
 }
+#endif
 
 #if defined(TOOLKIT_GTK)
 // static
